lib/gcloud/bigquery/connection.rb

```diff
@@ -462,14 +462,14 @@ def copy_table_config source, target, options = {}
           "configuration" => {
             "copy" => {
               "sourceTable" => {
```

Comment:
Makes sense. I would almost prefer that we pass the table string instead, but converting a hash to a string, and then a string to a hash would be silly choice for performance reasons.

The table does know API details in that it knows how to use the Google Client API object. It knows the project, dataset, and table are found in `@gapi["tableReference"]`, for example.